pkg/libvirttools/libvirt_domain.go

```diff
@@ -276,11 +287,62 @@ func (domain *libvirtDomain) SetVcpus(vcpus uint) error {
 	return domain.d.SetVcpusFlags(vcpus, libvirt.DOMAIN_VCPU_CONFIG|libvirt.DOMAIN_VCPU_LIVE)
 }
 
+// TODO: move this to a helper function file
+func determineNumberOfDeviceNeeded (memChangeInKib int64, isAttach bool) int {
+	var numberMemoryDevicesNeeded int
+
+	temp := math.Abs(float64(memChangeInKib) ) / float64(memoryDeviceSizeInKiB)
+	if temp < 1 {
+		if isAttach {
+			numberMemoryDevicesNeeded = 1
+		} else {
+			numberMemoryDevicesNeeded = 0
+		}
```

Comment:
Is this clause redundant?  It seems ceiling/floor in the "else" clause will do the same work.

Reply:
yes, you are right. will fix. thanks.